### PR TITLE
User model

### PR DESCRIPTION
**Pull request by JakeDish**

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const { Model, DataTypes } = require("sequelize");
+
```
Comment (ericnguyen23) [inline models/User.js:2]: Need to add/require bcrypt:
`const bcrypt = require('bcrypt');`

File: models/User.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    user_name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
```
Comment (ericnguyen23) [inline models/User.js:21]: add email validation:
```
validate: {
        isEmail: true,
      },
```

**Review by ericnguyen23 — CHANGES_REQUESTED** [review]

File: models/User.js (hunk 1 above)
Comment (JakeDish) [reply models/User.js:3]: Added

File: models/User.js (hunk 2 above)
Comment (JakeDish) [reply models/User.js:22]: Done!

**Review by ericnguyen23 — APPROVED** [review]

**Pull request merged**